src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -44,3 +44,4 @@
 [assembly: InternalsVisibleTo("Tester.SQLUtils")]
 [assembly: InternalsVisibleTo("Tester.ZooKeeperUtils")]
 [assembly: InternalsVisibleTo("AWSUtils.Tests")]
+[assembly: InternalsVisibleTo("ServiceBus.Tests")]
```

Comment:
Is this absolutely needed? One of @jason-bragg's goals when writing this provider was to make sure this adaptor could in theory be built outside Orleans repo entirely.